cnxepub/adapters.py

```diff
@@ -494,7 +494,11 @@ def _compute_shortid(ident_hash):
                 elif parent.metadata.get('version') is not None:
                     metadata['version'] = parent.metadata['version']
 
-            id_ = metadata.get('cnx-archive-uri') or child.attrib.get('id')
+            uuid_key = child.get('data-uuid-key')
```

Comment:
Just confirming that these changes only care about the presence of `data-uuid-key`, not the value of `data-uuid-key`.

It seems that the value is used a few lines above at line 442.

Reply:
Kinda. Although the reason I checked falseyness rather than noneness was because accepting `data-uuid-key` as empty string is probably also a formula for unhappiness.

Reply:
But if you mean present as in 'a string with positive length', then yes.

Reply:
I was just surprised that the value of `uuid_key` was pulled out of the element but not being used (it seems to be because the value is pulled out of the element in earlier code)

Reply:
ah, it is being used on line 500 to check for falseyness of the value